fast64_internal/oot/data/oot_actor_data.py

```diff
@@ -0,0 +1,43 @@
+class OoT_ActorData:
+    """Everything related to OoT Actors"""
+
+    from dataclasses import dataclass
+
+    @dataclass
+    class OoT_Actor:
+        id: str
+        key: str
+        name: str
+        category: str
+        tiedObjects: str
+
+    def __init__(self):
+        # Path to the ``ActorList.xml`` file
+        self.actorXML: str
+
+        # general actor list
+        self.actorList: list[self.OoT_Actor] = []
+
+        # list of tuples used by Blender's enum properties
+        self.ootEnumActorID: list[tuple] = []
+        self.ootEnumActorIDLegacy: list[tuple] = []  # for old blends
+
+        self.initActorLists()
```

Comment:
`initActorLists` is unconditionally called on `__init__` and unused elsewhere, so its code should just be inside `__init__` directly